### DISCUSSION TITLE
New model swin trasformer

### DISCUSSION THREAD

**Discussion by CesareDavidePace** [ultralytics/yolov5#10335] [Q&A]
hi,
I've seen templates on the web that use yolov5 templates with the swin transformer and I'd like to try them, but these use an old version of the code so there are problems when they run. If I want to use these modules in this version, do I just need to add the modules in the common.py?

**Comment by glenn-jocher** [comment]
@CesareDavidePace you can add any modules to models/common.py and then use these in custom model yamls.
